user_tools/docs/user-tools-onprem.md

```diff
@@ -0,0 +1,130 @@
+| **tools_jar**                  | Path to a bundled jar including RAPIDS tool. The path is a local filesystem                                                                                                                                                                                                                               | Downloads the latest rapids-tools_*.jar from mvn repo                                                                                                |     N    |
+| **filter_apps**                | Filtering criteria of the applications listed in the final STDOUT table is one of the following (`NONE`, `SPEEDUPS`). "`NONE`" means no filter applied. "`SPEEDUPS`" lists all the apps that are either '_Recommended_', or '_Strongly Recommended_' based on speedups.                                   | `SPEEDUPS`                                                                                                                                           |     N    |
+| **verbose**                    | True or False to enable verbosity to the wrapper script                                                                                                                                                                                                                                                   | False if `RAPIDS_USER_TOOLS_LOG_DEBUG` is not set                                                                                                    |     N    |
+| **rapids_options****           | A list of valid [Qualification tool options](../../core/docs/spark-qualification-tool.md#qualification-tool-options). Note that (`output-directory`, `platform`) flags are ignored, and that multiple "spark-property" is not supported.                                                                  | N/A                                                                                                                                                  |     N    |
+
+#### Use case scenario
+
+A typical workflow to successfully run the `qualification` command in local mode is described as follows:
+
+1. Store the Apache Spark event logs in local directory.
+2. User sets up the development machine:
+   1. configures Java
+   2. installs `spark_rapids_user_tools`
+3. User defines the onprem-cluster on which the Spark application were running. Note that the cluster does not have to be
+   active.
+4. The following script runs qualification tool locally:
+
+   ```
+   # define the wrapper cache directory if necessary
+   export RAPIDS_USER_TOOLS_CACHE_FOLDER=my_cache_folder
+   export EVENTLOGS=/LOGS_BUCKET/eventlogs/
```

Comment:
`LOGS_BUCKET`  ?

Reply:
I hadn't specified LOGS_BUCKET but it was meant to be path_to_eventlogs. Updated it to reflect the same